### PR TITLE
[Fleet] Document how to use with Kibana

### PR DESCRIPTION
**Pull request by nchaulet**
## Description 

As discussed with @aleksmaus  add documentation on how to run Fleet Server with Kibana as soon https://github.com/elastic/kibana/pull/86179 is merged.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,4 +16,47 @@ Either the key `a...` is used in the kibana config as this is the default:
...
+
+
+POST /_security/user/kibana_fleet_system
+{
+  "password" : "changeme",
+  "roles" : [ "kibana_system", "kibana_fleet_system" ]
+}
+```
+
+Than configure your Kibana with
```
Comment (aleksmaus) [inline README.md:57]: `Then`

**Review by aleksmaus — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -16,4 +16,47 @@ Either the key `a...` is used in the kibana config as this is the default:
...
\ No newline at end of file
+The alternative is to use `ES_SAVED_KEY` and pass it to fleet-server during setup with the value of the encryption key used in Kibana.
+
+
+### Kibana
+
+Currently there is some work to do to be able to run Kibana with Fleet Server and all the features are not yet supported, in the future, these workarounds will not be needed anymore.
+
+* Start fleet-server before Kibana, to create the mappings in ES.
+* Create and use a custom user as the `kibana_system` user
```
Comment (ruflin) [inline README.md:27]: For development, could I just pass in `elastic` / `changeme` and it would work too?

**Review by ruflin — APPROVED** [review]
Thanks for the docs.

File: README.md (hunk 2 above)
Comment (nchaulet) [reply README.md:27]: it's not possible to pass `elastic` as the kibana user this will be rejected

Comment (ruflin) [reply README.md:27]: Good to know.

**Pull request merged**